sympy/concrete/expr_with_limits.py

```diff
@@ -25,12 +26,12 @@ def _process_limits(*symbols):
     limits = []
     orientation = 1
     for V in symbols:
-        if isinstance(V, Symbol):
+        if isinstance(V, (Symbol, Idx)):
```

Comment:
@Upabjojr I am not very sure. If something like ``Sum(A[i], i)`` should work. Looks a little inconsistent in the way ``Sum`` usually works.
Ping @certik @asmeurer